doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -285,8 +285,8 @@ on the shape. In particular, the following geometries can be defined so far:</p>
 This operation supports two options:
 
 <ul>
-  <li><code>count</code>: when used, the total entities number is returned as a HTTP in the response name <code>X-Total-Count</code>.</li>
-  <li><code>cannonical</code>: when used, the response payload used the "cannonical form". See JSON entity representation format section
+  <li><code>count</code>: when used, the total number of entities is returned in the response as a HTTP header named <code>X-Total-Count</code>.</li>
```

Comment:
as you are changing this may be you can take the opportunity to introduce new lines when needed